source/docs/awards/portfolio.rst

```diff
@@ -105,11 +105,11 @@ Often teams will host events or outreaches with the intent of spreading the idea
 
 .. attention:: Appendix F of |gm1| contains **very specific definitions of words like Mentored, Started, Reached, Ran, and more**. Misuse of these words can be held against you in judging, so make sure you meet the definitions of terms if you use them.
 
-When documenting how many people an event reached, it can be tempting to ask the organizers for an official head count and use that number. However, |gm1| specifically details that simply being an event does not count someone as being reached by the team, they have to interact with your team in some way. Keeping a rough count of how many people you interact with can help you keep track of accurate numbers. **You may be questioned on how you know your numbers are accurate, so be ready to answer questions on how you kept track**.
+When documenting how many people an event reached, it can be tempting to ask the organizers for an official head count and use that number. However, |gm1| specifically details that simply being at an event does not count someone as being reached by the team, they have to interact with your team in some way. Keeping a rough count of how many people you interact with can help you keep track of accurate numbers. **You may be questioned on how you know your numbers are accurate, so be ready to answer questions on how you kept track**.
 
 Working with Other Teams
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Mentoring, starting, and helping other teams is critical to the program itself as well as some awards. These interactions should be documented in the portfolio, as well as the evidence that proves that you meet the definitions of Mentoring, Starting, and Assisting as outlined in |gm1|. In general, Mentoring requires regular, meaningful communication between teams, and Starting requires the team to recognize they were started by that team.
+Mentoring, starting, and helping other teams is critical to the program itself as well as some awards. These interactions should be documented in the portfolio, as well as the evidence that proves that you meet the definitions of Mentoring, Starting, and Assisting as outlined in |gm1|. In general, Mentoring requires regular, meaningful communication between teams, and Starting requires the new team to recognize they were started by that team.
```

Comment:
```suggestion
Mentoring, starting, and helping other teams is critical to the program itself as well as some awards. These interactions should be documented in the portfolio, as well as evidence that proves that you meet the definitions of Mentoring, Starting, and Assisting as outlined in |gm1|. In general, Mentoring requires regular, meaningful communication between you and a team, and Starting requires the new team agreeing that they were started by you.
```
In general, we try avoiding use `you`, but sometimes it simplifies statements significantly, and I think this is one of those cases.